src/common/components/ScrollBar/ScrollBar.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { ReactNode } from 'react'
+import SimpleBar from 'simplebar-react'
+import cn from 'classnames'
+
+// eslint-disable-next-line
+import 'simplebar/src/simplebar.css' //for some reason eslint was not accepting this
```

Comment:
I think you should use `import simplebar from 'simplebar/src/simplebar.css'` 

Please take a look at this sample https://github.com/pastelnetwork/pastel-electron-wallet/blob/master/src/features/receive/Receive.tsx (Line 12)

Reply:
@ngvtuan can you check here? https://www.npmjs.com/package/simplebar-react
![image](https://user-images.githubusercontent.com/71397124/120887433-cede4d00-c5c0-11eb-8138-273c2635369d.png)

Reply:
@ngvtuan can you check here? https://github.com/Grsmto/simplebar/blob/master/examples/react/src/App.js#L2-L4

Reply:
@dbizhev I think the problem here is you are using the wrong path of the CSS when you try to import it Please try `import '../simplebar/dist/simplebar.min.css' `

Reply:
@ngvtuan it's getting the error
![image](https://user-images.githubusercontent.com/71397124/120889313-7744df00-c5ca-11eb-923b-06340b1e1d01.png)

Reply:
@ngvtuan have you tested it on your end?

Reply:
@dbizhev let me check for the path. I think we need to include the `node_module` folder in the path. But It has another way to add the CSS file from the node_modules please take a look at here:  https://github.com/pastelnetwork/pastel-electron-wallet/blob/master/src/index.css (Line 31)

Reply:
@ngvtuan why does it need?

Reply:
@ngvtuan https://github.com/pastelnetwork/pastel-electron-wallet/pull/114/commits/90ddedfd8f8c7539ae36fa3945be2397859888c9#diff-85da366246340e911d7a0eb9153e64137a1b31f0686c74d9aef9ae4f032cb09eR9